apps/strapi-cms/Dockerfile

```diff
@@ -24,6 +24,7 @@ COPY --from=build /opt/node_modules ./node_modules
 WORKDIR /opt/app
 COPY --from=build /opt/app ./
 ENV PATH /opt/node_modules/.bin:$PATH
+RUN npm run configrestore
```

Comment:
I read the PR #81 and it says
> This command is to be ran on the ECS instance, before npm run start

It looks to me you are running the command during the build of Docker image.
Am I missing something?

Reply:
@simosalsi, tell me the exact line where to move the command

Reply:
Just before `CMD npm run start` should work if @kin0992 agrees

Reply:
command moved

Reply:
I don't think it will work; the run is a `image build step`. It is not executed at runtime

[RUN](https://docs.docker.com/engine/reference/builder/#run) is an image build step, the state of the container after a RUN command will be committed to the container image. A Dockerfile can have many RUN steps that layer on top of one another to build the image.

[CMD](https://docs.docker.com/engine/reference/builder/#cmd) is the command the container executes by default when you launch the built image. A Dockerfile will only use the final CMD defined. The CMD can be overridden when starting a container with docker run $image $other_command.

# Referece
https://stackoverflow.com/a/37462208

Reply:
"There can only be one CMD instruction in a Dockerfile. If you list more than one CMD, only the last one takes effect.". So how do we want to enter the execution of this configrestore script?